### PR TITLE
163 include open fee fines migration into migration tools close #163

### LINKED ISSUE #163: Include open fee-fines migration into migration_tools
More specifically, this migration task will deal with "manual fees/fines". This is the type that can be added on to a user record via the UI. They may or may not link to an item record. They are not, however, linked to loans.
image

To create manual fees/fines, you need to have configured Fee/fine owners and Manual charges>Fee/fine types
image

### PR DESCRIPTION
**Pull request by branchedelac**
Hi! Here's a mega PR! However it's mostly new code that only minimally touches the shared files.

For this to work we are going to need to upgrade the folio-uuid dependency https://pypi.org/project/folio-uuid/0.2.8/.

### CODE REVIEW

**Comment by branchedelac** [comment]
Out of scope:
https://github.com/FOLIO-FSE/folio_migration_tools/issues/616
https://github.com/FOLIO-FSE/folio_migration_tools/issues/617
https://github.com/FOLIO-FSE/folio_migration_tools/issues/618

File: src/folio_migration_tools/mapping_file_transformation/manual_fee_fines_mapper.py [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+                cache[match_value] = matching_record
+                return matching_record
+            else:
+                raise TransformationFieldMappingError(
+                    index_or_id, f"No matching {result_type} for {match_property}", match_value
+                )
+        else:
+            return cache[match_value]
+
+    def parse_date(self, folio_prop_name: str, index_or_id, mapped_value, legacy_object):
```
Comment (bltravis) [inline src/folio_migration_tools/mapping_file_transformation/manual_fee_fines_mapper.py:196]: @branchedelac Do we want to do any local timezone stuff here like we did with loans, or can we just assume UTC if no timezone indications is present in the source data?

Comment (branchedelac) [reply src/folio_migration_tools/mapping_file_transformation/manual_fee_fines_mapper.py:196]: https://github.com/FOLIO-FSE/folio_migration_tools/issues/619

Comment (bltravis) [reply src/folio_migration_tools/mapping_file_transformation/manual_fee_fines_mapper.py:196]: Were you thinking you'd include that in this PR, or create a new feature branch?

Comment (bltravis) [reply src/folio_migration_tools/mapping_file_transformation/manual_fee_fines_mapper.py:196]: Ok. I'll merge and we can move from there. I'm going to rebase, so you'll want to either wait to create the branch or rebase to main after this is merged.

**Comment by bltravis** [comment]
I went ahead and bumped the folio-uuid version as part of this PR. Once that is verified and merged, I'll merge.

**Pull request merged**